src/components/Modals/ExecuteModal.js

```diff
@@ -66,17 +66,22 @@ export const ExecuteModal = ({
   const rows = [...connected, ...disconnected].map((con) => ({
```

Comment:
If we would extract the `rows` building into a function we could add a test for this change, without having to go through setting up tests for the whole `ExecuteModal` component.

Reply:
Now that I've separated it into a function, it's now much more readable, and does not warrant a test IMO

Reply:
We should start to increase the coverage in remediation. Please add a test.

Reply:
If we extract the whole rows building into a function outside the component and add a test for it, one asserting being it generates the right status it is a start to improve the coverage with just a tiny bit more effort.

Reply:
I added a test which is a bit more E2E, is it okay?